docs/status.md

```diff
@@ -87,7 +87,7 @@ one of the following tracking labels.
 | dynamic_slice            | yes           | yes          | yes            | yes             | no          |
 | dynamic_update_slice     | yes           | yes          | yes            | yes             | no          |
 | einsum                   | no            | revisit      | no             | yes             | no          |
-| exponential              | yes           | yes          | yes            | yes             | no          |
+| exponential              | yes           | yes          | yes            | yes             | yes         |
```

Comment:
Should we also update TypeInference.cpp with `/*exponential_c1*/` next to `INFER_RETURN_TYPE_COMPONENTS_FROM_OPERANDS(ExpOp)`?

Reply:
Good catch! I was thinking of this but not sure how to add a comment for all INFER_RETURN_TYPE_COMPONENTS_FROM_OPERANDS macros.
Take your suggestion that add this for ExpOp only in this specific PR.

Reply:
I changed my mind: I noticed that all ops with `INFER_RETURN_TYPE_COMPONENTS_FROM_OPERANDS` has `HLO_CompatibleOperandsAndResultType` in ODS thus will have xyz_c1 here. So I believe there is no need to add another /*exponential_c1*/ after the micro. 
What do you think of it?

Reply:
I also think that adding comment on `HLO_CompatibleOperandsAndResultType` is sufficient?

Reply:
Agreed. Let Eugene continue the review.

Reply:
Agreed that it's sufficient.